pmpro-membership-manager-role.php

```diff
@@ -17,6 +17,7 @@ function pmprommr_setup_role() {
         'pmpro_memberships_menu' => true,
 		'pmpro_dashboard' => true,
 		'pmpro_edit_memberships' => true,
```

Comment:
We've always been giving this cap. It's used in includes/rest.php. It's a little unclear what it really means. This plugin might check that cap itself... or the add member from admin add on?

Maybe the rest.php should really be checking for the pmpro_membershiplevels cap. Or this was an early version of pmpro_edit_members and we should start using that now?

Could be okay to keep it, but should at least add a comment here, especially if we change behavior in includes/rest.php.